fractions/javaee/jpa/pom.xml

```diff
@@ -51,6 +51,11 @@
       <groupId>io.thorntail</groupId>
       <artifactId>ee</artifactId>
     </dependency>
+    <!-- WF14 -->
```

Comment:
There's a hard dependency on Infinispan fraction for JPA? That's not good

Reply:
This is because of https://github.com/wildfly/wildfly/blob/14.0.1.Final/feature-pack/src/main/resources/modules/system/layers/base/org/jboss/as/jpa/main/module.xml#L49 I believe

Reply:
The Hibernate+Infinispan integration has moved from Hibernate to Infinispan, that might be the reason. But I'll look deeper.

Reply:
Unfortunately it seems it really is required. Otherwise I'm getting

```
    "WFLYCTL0412: Required services that are not installed:" => ["org.wildfly.clustering.infinispan.cache-configuration.hibernate.entity"],
    "WFLYCTL0180: Services with missing/unavailable dependencies" => [
        "jboss.persistenceunit.\"CdiJpaJtaTest.jar#MyPU\".__FIRST_PHASE__ is missing [org.wildfly.clustering.infinispan.cache-configuration.hibernate.entity]",
        "jboss.persistenceunit.\"CdiJpaJtaTest.jar#MyPU\" is missing [org.wildfly.clustering.infinispan.cache-configuration.hibernate.entity]"
    ]
```

That is, the persistence unit service (from the JPA subsystem) depends on the Hibernate cache services (from the Infinispan subsystem).

Not sure how it worked before.

Reply:
Well that's going to blow up uber jar sizes.

Oh well

Reply:
It does :-( The `jpa-jaxrs-cdi-jta` example has 110 MB with 2.2.1 and 126 MB now.